ds3-sdk/src/main/java/com/spectralogic/ds3client/commands/spectrads3/CancelVerifyOnAllTapesSpectraS3Request.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * ******************************************************************************
+ *   Copyright 2014-2015 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+// This code is auto-generated, do not modify
+package com.spectralogic.ds3client.commands.spectrads3;
+
+import com.spectralogic.ds3client.networking.HttpVerb;
+import com.spectralogic.ds3client.commands.AbstractRequest;
+
+public class CancelVerifyOnAllTapesSpectraS3Request extends AbstractRequest {
+
+    // Variables
+    
+    // Constructor
+    
+    public CancelVerifyOnAllTapesSpectraS3Request() {
+        
+        this.getQueryParams().put("operation", "cancel_verify");
+    }
+
+
+    @Override
+    public HttpVerb getVerb() {
+        return HttpVerb.PUT;
+    }
+
+    @Override
+    public String getPath() {
+        return "/_rest_/tape";
+    }
+    }
```

Comment:
I'm not going to reject the pull request for this, but it seems the indentation here is off.